doc/changelog/02-specification-language/18590-cleanup_16230.rst

```diff
@@ -0,0 +1,9 @@
+- **Changed:**
+  warnings `future-coercion-class-constructor`
+  and `future-coercion-class-field` about ``:>`` in :cmd:`Class` as
+  error by default. This offers a last opportunity to replace ``:>``
+  with ``::`` (available since Coq 8.18) to declare typeclass instances
+  before making ``:>`` consistently declare coercions in all records in
+  next version
```

Comment:
I wrote a script to do this replacement in f-c-l (PR/overlay at https://github.com/mit-plv/fiat-crypto/pull/1827), maybe it's worth linking to https://gist.github.com/JasonGross/59fc3c03664f2280849abf50b531be42 here?

Reply:
@JasonGross thanks for the overlay! And thanks for the script, I added it, along with the one I used for iris, to the changelog.